features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_router_bootstrap.c

```diff
@@ -1407,6 +1407,7 @@ static void thread_address_registration_tlv_parse(uint8_t *ptr, uint16_t data_le
                 //Register GP --> 16
                 int retVal = thread_nd_address_registration(cur, tempIPv6Address, mac16, cur->mac_parameters->pan_id, mac64, &new_neighbour_created);
                 thread_extension_address_registration(cur, tempIPv6Address, mac64, new_neighbour_created, retVal == -2);
+                (void) retVal; // quite compiler in case thread_extension_address_registration is ((void) 0)
```

Comment:
Just leave out the comment field.
`(void)arg` is quite used idiom, so the intention should be clear to anyone already.

And any way, I think you meant ` // silence compiler warning in case ...`